gtest/src/wasm_executor.rs

```diff
@@ -208,24 +238,32 @@ mod meta_tests {
             "../target/wasm32-unknown-unknown/release/demo_meta.wasm",
         );
 
-        let result: Vec<Wallet> = program.meta_state(&Some(Id {
-            decimal: 2,
-            hex: vec![2u8],
-        }));
+        let result: Vec<Wallet> = program
+            .meta_state(&Some(Id {
+                decimal: 2,
+                hex: vec![2u8],
+            }))
+            .expect("Meta_state failed");
 
         assert_eq!(result.encode(), vec![0]);
```

Comment:
no need in encoding here. compare Vec<Wallet> types

Reply:
Encoding removed.